src/measurement_buffer.cpp

```diff
@@ -150,53 +129,46 @@ void MeasurementBuffer::BufferPCLCloud(const \
       return;
     }
 
-    point_cloud_ptr cld_global(new pcl::PointCloud<pcl::PointXYZ>);
-    geometry_msgs::TransformStamped tf_stamped = _buffer.lookupTransform(_global_frame, cloud.header.frame_id.c_str(), pcl_conversions::fromPCL(cloud.header).stamp);
-    tf::StampedTransform tf_transform;
-    tf::transformMsgToTF(tf_stamped.transform, tf_transform);
-    tf_transform.frame_id_ = _global_frame;
-    tf_transform.stamp_ = tf_stamped.header.stamp;
-    pcl_ros::transformPointCloud(cloud, *cld_global, tf_transform);
-    cld_global->header.stamp = cloud.header.stamp;
+    // transform the cloud in the global frame
+    point_cloud_ptr cld_global(new sensor_msgs::PointCloud2());
+    geometry_msgs::TransformStamped tf_stamped = _buffer.lookupTransform(_global_frame, cloud.header.frame_id, cloud.header.stamp);
```

Comment:
try to keep the line lengths inside the guards.

Reply:
Ok, changed it manually. Do you use any auto formatting tools that you would recommend to keep consistency with your code?

Reply:
I don't -- frankly I'm not too picky as long as its consistent and that's the style I had been working with prior. Generally my philosophy is that it should stay within a 100 line guard when reasonable, and any subsequent line extensions should be aligned with that end guard as well.